src/ai/backend/manager/models/utils.py

```diff
@@ -362,3 +362,14 @@ def agg_to_array(column: sa.Column) -> sa.sql.functions.Function:
 
 def is_db_retry_error(e: Exception) -> bool:
     return isinstance(e, DBAPIError) and getattr(e.orig, "pgcode", None) == "40001"
+
+
+def description_msg(version: str, detail: str | None = None):
+    val = f"Added since {version}."
+    if detail:
+        val = f"{val} {detail}"
+    return val
+
+
+def deprecation_reason_msg(version: str):
```

Comment:
Let's add the `detail: str`to here alos.

Reply:
Except for this, everything looks good to me.

Reply:
Updated it, thank you!